Sources/MapboxCoreNavigation/EHorizon/EHorizonEdge.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct EHorizonEdge {
+
+    /** Unique identifier of the directed edge */
+    public let id: UInt
```

Comment:
In Swift, ID is always spelled out as “identifier”. We should probably also make a type alias for ID-like properties for forward compatibility and a convenient place to document any limitations on the numeric values in the future.